README.md

```diff
@@ -1 +1,2 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+You do not need any pre requisites nor any instructions for this project. This is a basic project to get us started with html, css, git, GitHub, and linters.
```

Comment:
- Hey, great job with this project so far, I admire your energy, however, the purpose of this project is to help you get familiar with setting up a professional GitHub repository and get comfortable in doing them repetitively
   - Let's improve our readme by using this [template](https://github.com/microverseinc/readme-template) but you should customize it to suit this current project's needs. 
   - Replace all the placeholder text as they're there meant to guide on how to add some content in the readme
   > Follow the template closely to update them with your personal content but maintain its structure and format.